_toc.yml

```diff
@@ -177,8 +177,10 @@ entries:
     entries:
     - file: docs/platform/howto/monitoring-services
     - file: docs/platform/concepts/logs-metrics-alerts
+    - file: docs/tutorials/anomaly-detection
+      title: Streaming anomaly detection with Apache Flink, Apache Kafka and PostgreSQL
```

Comment:
Any idea how to make it shorter? this shows in the docs sidebar actually

Reply:
good point!

Reply:
```suggestion
      title: Streaming anomaly detection
```